myapp/tests/test_stock_api.py

```diff
@@ -31,6 +31,10 @@ def test_get_stock_historic_prices(self):
             self.assertIsInstance(response, list)
 
         # testing time range
+        response = get_stock_historic_prices(self.existed_symbols[0], '1y')
+        first_year = datetime.fromisoformat(response[0].get('date')).year
+        second_year = datetime.today().year
+        self.assertEquals(second_year - first_year, 1)
```

Comment:
yeah, this just a basic test on the cloud service, we would extend on need.
I added this basic testing because this feature is already implemented in our code, in the `stock_api.py` file